code/src/components/app-range-slider/AppRangeSlider.scss

```diff
@@ -12,7 +17,13 @@
     }
   }
 
+  &__range-container {
+    display: flex;
+  }
+
   &__range {
+    width: 95%; // cut width because thumbs trigger horizontal scrollbar
```

Comment:
use `px` instead `%`
We decided to change it

Reply:
Take a look:

https://github.com/user-attachments/assets/1da44173-d8e8-4809-9259-8ce6aaa56dbc

My solution is related to mui's broken styling. I did it because the component should be reusable and do not trigger horizontal scroll